core/input.ts

```diff
@@ -90,7 +90,8 @@ export class Input {
    *     field again.  Should be unique to the host block.
    * @returns The index following the last inserted field.
    */
-  insertFieldAt(index: number, field: string|Field, opt_name?: string): number {
+  insertFieldAt(index: number, field: string|Field<unknown>, opt_name?: string):
```

Comment:
This should also be converted to a generic method, since it's public.

Reply:
Done!